### PR TITLE
Improve texts

### LINKED ISSUE #609: Text review in new Wireless configurator of Thymio Suite
In the new configuration tools, I propose to do some text modifications:

### In the tools of Thymio Suite
image

- [ ] "Pair a Wireless Thymio to a Wireless dongle" 
- [ ] "Pair a group of Wireless Thymio robots independently"

### On the pop-up window before the configurator
image

- [x] "Open the Wireless Thymio network configurator"
- [x] "All Wireless Thymio robots associated to the Wireless dongle plugged on this computer will be disconnected until the Wireless Thymio network configuration window is closed."

### In the Pair a Thymio Wireless to a dongle interface
image

- [x] "Wireless Thymio network configurator"
- [x] "Plug the Wireless Thymio robot you want to pair using the USB cable"
- [x] "Plug the Wireless dongle"
- [x] "Please plug a Wireless Thymio robot with a USB cable to this computer"

### In the Advanced mode interface
image
- [x] "Wireless Thymio network configurator"
- [x] "To pair multiple Wireless Thymio robots to one Wireless dongle:"
- [x] "03. Click on Pair !"

### In the pair a case of thymios interface
image

- [x] "Wireless Thymio network configurator"
- [x] "Plug the Wireless Thymio robot you want to pair using the USB cable"
- [x] "Plug the Wireless dongle"
- [x] "Tip: Put a sticker dot of the same color on both the Wireless Thymio robot and the Wireless dongle to identify them!
(open question, should we advise the color dot sticker or the serial number of the robot? could depends also of next move in production)

### PR DESCRIPTION
**Pull request by cor3ntin**
Fixes #609

### CODE REVIEW

File: aseba/launcher/src/qml/SettingsMenu.qml [hunk 1]
```diff
@@ -16,12 +16,12 @@ Rectangle {
...
 
         ListElement {
-            name: qsTr("Pair a Thymio Wireless to a Dongle")
+            name: qsTr("Pair a Wireless Thymio to a Wireless dongle")
             action: "thymio2-pairing"
         }
 
         ListElement {
-            name: qsTr("Pair a case of thymios")
+            name: qsTr("Pair a case of Wireless Thymios")
```
Comment (mbonani) [inline aseba/launcher/src/qml/SettingsMenu.qml]: Is it more correct "Pair a Wireless Thymio case" ?

**Comment by cor3ntin** [comment]
> "Pair a case of Wireless Thymios" => "Pair a Wireless Thymio case"
Not sure about this one

**Comment by mbonani** [comment]
image

it seems more correct in this sense

**Comment by cor3ntin** [comment]
Are we pairing the case or individual robots in that case?

**Comment by mbonani** [comment]
we pair robot in that case

**Pull request merged**